nni/compression/pytorch/speedup/jit_translate.py

```diff
@@ -83,243 +86,17 @@ def parse_constant(cvalue, speedup):
 
     inputs = op_node.inputs()
     input_values = [parse_constant(_i, speedup) for _i in inputs]
-    func = trans_from_jit_to_python[op_node.kind()](op_node, speedup)
+    func = trans_func_dict[op_node.kind()](op_node, speedup)
```

Comment:
add an error handling here may be better for debugging, if `op_node.kind()` not in `trans_func_dict`

Reply:
ok